### PR TITLE
Require all checks for mergify

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ pull_request_rules:
     conditions:
       - "status-success=ci/circleci: build"
       - "status-success=ci/circleci: noauth"
+      - "status-success=ci/circleci: install-minikube"
+      - "status-success=ci/circleci: integration-old-kind"
+      - "status-success=ci/circleci: integration-presubmit-kind"
```
Comment (linsun) [inline .mergify.yml]: @howardjohn is this test stable?  your PR failed on it.

Comment (howardjohn) [reply .mergify.yml]: It was stable, but something change in istio/istio causing the failure, I am looking into it now.

**Comment by costinm** [comment]
We really need a test in istio/istio that clones istio/installers and runs one of those tests.

**Comment by linsun** [comment]
ok to merge this now?  assume you'll get all failed 2 tests passing

**Comment by howardjohn** [comment]
@linsun rebasing to make sure test pass, ill merge when they do

**Pull request merged**